#!/usr/bin/python3
def raise_exception_msg(message=""):
    # raise NameError with a message
    raise NameError(message)
